pipelines/common/pom.xml

```diff
@@ -251,6 +251,19 @@
       <artifactId>beam-sdks-java-io-google-cloud-platform</artifactId>
       <version>${beam.version}</version>
     </dependency>
+    <dependency>
+      <groupId>org.junit.jupiter</groupId>
+      <artifactId>junit-jupiter-api</artifactId>
+      <version>5.10.1</version>
```

Comment:
Can we have the latest versions to be used over here, same for `mockito-junit-jupiter` as well.

Reply:
Resolved!